gettsim/policy_environment.py

```diff
@@ -272,7 +274,7 @@ def _parse_kinderzuschl_max(date, params):
 
     """
 
-    if date.year >= 2021:
+    if date.year >= 2024 and 2023 > date.year >= 2021:
```

Comment:
```suggestion
    if (date.year >= 2024) or (2023 > date.year >= 2021) :
```

Brackets just for clarity. But the `or` instead of `and` is important.